src/chlorine/core.cljs

```diff
@@ -20,7 +20,12 @@
         (.. js/atom -workspace
             (observeTextEditors subscribe-editor-events))))
 
+(defn- install-dependencies-maybe []
+  (-> (.install (js/require "atom-package-deps") "chlorine")
+      (.then #(.log js/console "All dependencies installed."))))
```

Comment:
What about using the `atom/info` instead?

Reply:
Yes, that seems better :)